pkg/storaged/block/format-dialog.jsx

```diff
@@ -532,6 +544,17 @@
                     }
                 }
 
+                if (type == "swap") {
+                    config_items.push(["fstab", {
+                        dir: { t: 'ay', v: encode_filename("none") },
+                        type: { t: 'ay', v: encode_filename("swap") },
+                        opts: { t: 'ay', v: encode_filename(mount_now ? "defaults" : "noauto") },
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19843-20240116-090759-3ab49c70-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)